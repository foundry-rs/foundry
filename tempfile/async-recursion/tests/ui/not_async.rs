use async_recursion::async_recursion;

#[async_recursion]
fn not_async() {}

fn main() {}