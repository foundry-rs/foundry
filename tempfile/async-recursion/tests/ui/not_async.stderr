error: expected an async function
 --> tests/ui/not_async.rs:3:1
  |
3 | #[async_recursion]
  | ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `async_recursion` (in Nightly builds, run with -Z macro-backtrace for more info)
